spec/javascripts/support/jasmine_helper.rb

```diff
@@ -63,7 +63,7 @@ def call(env)
   end
 
   config.chrome_cli_options = {
-    'headless' => nil,
+    'headless' => 'new',
```

Comment:
Do we need to do anything to ensure local/dev/ci get the versions of chrome with this feature?  For example, dev on ubuntu/debian/older centos/rhel.

Reply:
Can we check for it? Or enforce it somewhere?

Reply:
Great question - I'll dig into that.

Reply:
We'll look at this in a followup.

Reply:
https://github.com/ManageIQ/manageiq-ui-classic/pull/8665